tests/Circle.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Circle from '../src/Circle';
+
+const renderWithProps = (props = {}, options) => render(<Circle { ...props } />, options);
+
+describe('Circle Component', () => {
```

Comment:
What about testing if the `onTransitionEnd` callback is being properly fired when passed?

Reply:
Ok, I was trying to do this but didn't manage to do it. How can I fire the `transitionend` event? I tried creating a CustomEvent with the correct name but it didn't work 🙁

Reply:
You can just do:

```js
const circleElem = ...

fireEvent.transitionEnd(circleElem); // It will fire 

// make assertion here to check whether `onTransitionEnd` callback was called properly or not
```

Please note that you must pass to `fireEvent.transitionEnd()` the node element that has `onTransitionEnd` attached. Which in this case is the `<circle>`